src/cmd/script.rs

```diff
@@ -45,109 +45,108 @@ pub fn new<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true)
-                .multiple(true),
+                .multiple_values(true),
         )
         .arg(
-            Arg::with_name("only-defines")
+            Arg::new("only-defines")
                 .long("only-defines")
                 .help("Only output commands to define macros (Vivado only)"),
         )
         .arg(
-            Arg::with_name("only-includes")
+            Arg::new("only-includes")
                 .long("only-includes")
                 .help("Only output commands to define include directories (Vivado only)"),
         )
         .arg(
-            Arg::with_name("only-sources")
+            Arg::new("only-sources")
                 .long("only-sources")
                 .help("Only output commands to define source files (Vivado only)"),
         )
         .arg(
-            Arg::with_name("no-simset")
+            Arg::new("no-simset")
                 .long("no-simset")
                 .help("Do not change `simset` fileset (Vivado only)"),
         )
         .arg(
-            Arg::with_name("vlogan-bin")
+            Arg::new("vlogan-bin")
                 .long("vlogan-bin")
                 .help("Specify a `vlogan` command")
                 .takes_value(true)
-                .multiple(false)
+                .multiple_values(false)
                 .default_value("vlogan")
                 .number_of_values(1),
         )
         .arg(
-            Arg::with_name("vhdlan-bin")
+            Arg::new("vhdlan-bin")
                 .long("vhdlan-bin")
                 .help("Specify a `vhdlan` command")
                 .takes_value(true)
-                .multiple(false)
+                .multiple_values(false)
                 .default_value("vhdlan")
                 .number_of_values(1),
         )
         .arg(
-            Arg::with_name("no-abort-on-error")
+            Arg::new("no-abort-on-error")
                 .long("no-abort-on-error")
                 .help("Do not abort analysis/compilation on first caught error (only for programs that support early aborting)")
         )
         .arg(
-            Arg::with_name("compilation_mode")
+            Arg::new("compilation_mode")
                 .long("compilation-mode")
                 .help("Choose compilation mode for Riviera-PRO option: separate/common (Riviera-PRO only)")
                 .takes_value(true)
-                .multiple(false)
-                 .default_value("common")
+                .default_value("common")
                 .possible_values(&[
                     "separate",
                     "common",
                 ])
         )
         .arg(
-            Arg::with_name("package")
-                .short("p")
+            Arg::new("package")
+                .short('p')
                 .long("package")
                 .help("Specify package to show sources for")
                 .takes_value(true)
-                .multiple(true)
+                .multiple_occurrences(true)
```

Comment:
What about `multiple_values(true)`?

Also below for the `exclude` argument.

Reply:
I would suggest to keep this as is, forcing users to explicitly indicate individual packages with `-p package1 -p package2` (similar to the now updated target). Same for below.

Reply:
Okay, but this is a breaking change, right? If so, please mention it in the Changelog.

Reply:
Ah no, we had `number_of_values(1)` before. Okay, then it does not change for the user.